components/TabList/tab-list.tsx

```diff
@@ -37,13 +37,14 @@ const TabList = ({ tabList, pageId, selectedTab }: TabListProps) => {
           role="tab"
           aria-selected={selectedTab === tab.name.toLowerCase() ? "true" : "false"}
           data-state={selectedTab === tab.name.toLowerCase() ? "active" : "inactive"}
-          tabIndex={-1}
+          tabIndex={0} // Added tabIndex for accessibility
```

Comment:
The `<nav />` houses links, as it should, so although visually this looks like tabs, they really aren't.

```suggestion
```

We can also remove the `role="tablist"` from the `<nav />`.